cmd/cli/plugin/package/test/package_plugin_integration_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package tkgpackageclient_test
+package main
```

Comment:
Question: Why is the test package renamed to main?

Reply:
Cause I moved it to the ```main``` package of ```cmd/cli/plugin/package/test``` from ```pkg/v1/tkg/test/packageclients``` and the package name needed to match  ```cmd/cli/plugin/package/test``` and wouldn't accept multiple package names.